### DISCUSSION TITLE
Build from source on Android

### DISCUSSION THREAD

**Discussion by janicduplessis** [software-mansion/react-native-reanimated#1830] [Ideas]
Currently source is not shipped on Android, I think it would be nice to have a way to compile from source, similar to what is possible to do with react-native.

- Ship source as part of npm package alongside prebuilt artifacts
- Add docs on how to configure source build

Source builds make it easier to debug issues as well as test fixes directly in apps.

**Comment by mrousavy** [comment]
I have created a PR for this here: https://github.com/software-mansion/react-native-reanimated/pull/1837
